docs/roomHandlers/fishingRoom.md

```diff
@@ -0,0 +1,38 @@
+# Fishing Room
+
+A room with a beach and a boat dock. The player can fish from dock or leave. Fishing may result in catching a fish.
+
+## States
+
+- **AtEntrance**: The player is on a dock with a fishing pole. Total number of fish caught is X so far!
```

Comment:
Due to the following transitions, I think you also need to keep track of the state of the fishing pole and output if it is broken or not

```
BrokenLine --> AtEntrance: stop
BrokenLine --> AtEntrance: fix
```

Reply:
Updated the output.